### ISSUE TITLE
Chicken (animal)

### ISSUE THREAD

**Issue by wkonkel** [Templarian/MaterialDesign#3455]
There are already icons for: dog, cat, fish, panda, pig, cow, penguin (linux), elephant, and rabbit. There should be a chicken icon too!

Something along the lines of:

![image](https://user-images.githubusercontent.com/26543/45195970-ecfba080-b20e-11e8-8d14-95c4342e70d4.png)

**Comment by alwashe** [comment]
no chicken?

**Comment by thom13** [comment]
Please for my chicken-coop...

**Comment by realbuz** [comment]
I also want a chicken icon for my coop please !!

**Comment by Hersh3yy** [comment]
me three!

**Comment by mloebl** [comment]
Just found this, adding a + to this too as monitor our chicken coop, so would love a generic chicken icon vs a "bird" or a food item

**Comment by Swiftnesses** [comment]
+1 for this.

**Comment by tankdeer** [comment]
+1 here too

**Comment by wakawakapogsSr** [comment]
Chicken dinner. We are waiting

**Comment by Templarian** [comment]
Really should get this added, but don't have a design. Added the high priority label to get more eyes. 🙃

**Comment by andrejsharapov** [comment]
maybe
![image](https://user-images.githubusercontent.com/44722574/151188380-40c581b2-43b4-44f5-8de6-1b9b0aa6d321.png)

<details>
<summary>-</summary>

![trash](https://user-images.githubusercontent.com/44722574/151188413-408e062c-5ce4-4399-a88e-df28732d5657.png)

</details>

**Comment by Hersh3yy** [comment]
Tail is rather ducky, needs to be more chickeny. See: https://media.istockphoto.com/photos/chicken-view-from-the-back-tail-picture-id1008807532

Better than nothing already btw! So happy this is going (flying) somewhere

**Comment by andrejsharapov** [comment]
![image](https://user-images.githubusercontent.com/44722574/151247103-10009e26-e105-4c58-a72b-de0f1aa38556.png)

**Comment by mloebl** [comment]
That second one looks a lot better to me!

**Comment by andrejsharapov** [comment]
Yes, but unfortunately, at 24px it does not look very understandably )

**Comment by andrejsharapov** [comment]
v3 it's harder than it looks)

![image](https://user-images.githubusercontent.com/44722574/151253608-a0a80b60-6c4d-4f96-b0c9-1a5801ad6be2.png)

**Comment by snis** [comment]
![chicken](https://user-images.githubusercontent.com/205149/152656992-2fcb33cc-af10-4ab7-9d83-1abf6ff9cb98.svg)

**Comment by Templarian** [comment]
@andrejsharapov I like the idea of modifying the turkey as it keeps the MD design.

@snis A side view will be ideal for chicken, but a face view works nice for Chicken also. Might need to exaggerate the 3 circles above the head.

**Comment by andrejsharapov** [comment]
I had to make the tuft a little larger so that at 24px it was clear that it was a chicken))

![image](https://user-images.githubusercontent.com/44722574/152658684-e4b8595d-7166-4aad-9ca7-493b1ad10f55.png)
[chicken.zip](https://github.com/Templarian/MaterialDesign/files/8008900/chicken.zip)

**Comment by snis** [comment]
> @andrejsharapov I like the idea of modifying the turkey as it keeps the MD design.
> 
> @snis A side view will be ideal for chicken, but a face view works nice for Chicken also. Might need to exaggerate the 3 circles above the head.

@andrejsharapov I like your icon, just trying different aspects.

@Templarian Sideview it is ;)

![chicken-profile2](https://user-images.githubusercontent.com/205149/152660020-963a36fe-097b-41f3-866e-59969bef6339.svg)

**Comment by JIK12345** [comment]
Chicken icon would be great for my automatic chicken coop door! Is this going to be added??

**Comment by averater** [comment]
I would also like this and very nice suggestions above but seems to not happen much?

**Comment by Templarian** [comment]
![chicken](https://github.com/Templarian/MaterialDesign/assets/338885/fdf7bc77-5a20-48a1-b3b3-19770c19730d)

More tweaks for the chicken head, but adding the chicken body.

**Comment by jwboulet** [comment]
[chicken.svg.zip](https://github.com/Templarian/MaterialDesign/files/14889694/chicken.svg.zip)
Just the head...

**Comment by andrejsharapov** [comment]
I see this has already been [added](https://pictogrammers.com/library/mdi/history/). Only the author is listed incorrectly. 
Most likely there will not be a separate head))

**Comment by Xanantec-James** [comment]
would it be possible to add a multiple version of this as well? something along the lines of [account-multiple](https://pictogrammers.com/library/mdi/icon/account-multiple/) but with the chicken icon
